portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,13 @@ function loadSkills () {
  */
  async function fetchFromData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('secret-message').innerText = quote; 
+  // This gives a list of messages
+  const msgs = await response.json();
+  console.log(msgs); 
+
+  // Adds all of the messages to an unordered list
+  msgs.forEach(m => {
+    const node = `<p class="msg">${m}</p>`; 
```

Comment:
This is a string instead of DOM node. Does it work if you directly append a string? My understanding is `<p>` will show up as escaped text.

Reply:
yea it shows up like it does in the picture -- would it be better to make a node with createElement?

Reply:
Umm I wonder jquery does something to process this. But anyway I think it is better to make a node with createElement.

Reply:
ok fixed :)

Reply:
hopefully it's okay haha